experiment/measurer/experiment_coverage_utils.py

```diff
@@ -0,0 +1,189 @@
+# Copyright 2020 Google LLC
```

Comment:
Really awesome this is in its own module.

Could you please think of a better name? `experiment_coverage_utils.py` doesn't really say much to me other than this is not `coverage_utils.py`.

Reply:
Not very good at naming, either. My suggestions:
* `coverage_printer.py`
* `coverage_data_printer.py`
* `data_to_csv_utils.py`

Reply:
Maybe coverage_over_time_utils?

Reply:
Or maybe detailed_coveage_utils?